BuildHelpers/Public/Get-ModuleFunctions.ps1

```diff
@@ -41,19 +43,19 @@ function Get-ModuleFunctions {
         $params = @{
             Force = $True
             Passthru = $True
-            Name = $Name
+            Name = Get-FullPath $Name
```

Comment:
Same here - I assume this will always be a path, but comment based help could use a tweak to avoid confusion (i.e. it's not either a name or path, it's a path and only a path)?

Reply:
I don't follow.
CBH on `Get-FullPath`?
Or using a named parameter instead of positioned?

Reply:
Oh sorry!  For all of these, they're used with `Import-Module -Name` - i.e. could be either a Path, or a Name.  It really should always be a path in the context of buildhelpers, but comment based help for `Name` still mentions it: `Name or path to module to inspect.`